core/src/main/java/io/undertow/client/ajp/AjpClientConnection.java

```diff
@@ -239,7 +245,7 @@ private void initiateRequest(AjpClientExchange AjpClientExchange) {
 
         String connectionString = request.getRequestHeaders().getFirst(CONNECTION);
         if (connectionString != null) {
-            if (CLOSE.equalToString(connectionString)) {
+            if (connectionString.equals(CLOSE)) {
```

Comment:
This is wrong. CLOSE is a HttpString and connectionString is a String

Reply:
Whoops, I forgot a line of code. Thanks for the review.